packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -68,6 +67,12 @@ export interface TooltipProps
   /** Defines whether tooltip is displayed. */
   open?: boolean
 
+  // TODO
+  onOpenChange?: (
+    e: React.MouseEvent | React.FocusEvent | React.KeyboardEvent,
+    data: TooltipProps,
+  ) => void
+
   /**
    * TODO: should this be centralized?
```

Comment:
Yes please

Reply:
I don't know what this means as it was added previously 🤔 I removed it for now